### ISSUE TITLE
feature - Budget report in bar graph

### ISSUE THREAD

**Issue by brianl44** [moneymanagerex/moneymanagerex#2130]
Could a budget report by created for categories that displays as bar graphs for estimated and actual. Then if you click the category it would drop down into a subcategories chart.

**Comment by vomikan** [comment]
I suggest to replace radar bar to Line Column

![image](https://user-images.githubusercontent.com/6836805/106357162-6d0d8e00-6315-11eb-846b-fffeee92ccc2.png)

https://apexcharts.com/vue-chart-demos/mixed-charts/line-column/

**Comment by vomikan** [comment]
@whalley what do you think about this issue?

**Comment by whalley** [comment]
Yes that should be possible, would need to have -ve and +ve vertical axes. Assign to me and I’ll have a think about adding it to the “Budget Performance” report.

**Comment by vomikan** [comment]
I thought that if the values differ by a couple of orders of magnitude, then the graph may not be informative.

**Comment by vomikan** [comment]
@whalley could you make one graph for all categories?

**Comment by whalley** [comment]
Just to be clear:
1. You want the bars to represent the 'estimated' budget values (stacked for each sub-catgeory)
2. You want the line to represent the 'actual' for ALL the combined sub-categories.
3. This will appear as an optional chart on the 'Budget Performance' report? -- This would seem to most appropriate report.

Is that correct

**Comment by vomikan** [comment]
I would prefer a single report, looking at which you can estimate the entire budget at once.
I would prefer №2. 
The actual funds spent/earned should be displayed in a column. One column per category.
The line must be at a height proportional to the planned amount.
If the line intersects the column, then the budget for this category is violated. If all the columns are below the line, then everything is OK.

**Comment by whalley** [comment]
OK, understood. Will see what I can do :-)
